.github/CODEOWNERS

```diff
@@ -6,7 +6,11 @@
 # the repo. Unless a later match takes precedence,
 # @openconfig/featureprofiles-maintainers will be requested for
 # review when someone opens a pull request.
-*       @openconfig/featureprofiles-maintainers @openconfig/featureprofiles-quattro
+*       @openconfig/featureprofiles-maintainers @openconfig/featureprofiles-quattro-tl
+
+# reviewers for tests which are ported from ate-tests to 
+# otg-tests may be reviewed by this team
+**/otg-tests/**    @openconfig/featureprofiles-maintainers-otg
```

Comment:
`@openconfig/featureprofiles-maintainers-otg` is noted not to exist.

Reply:
Fixed - added permissions for the team to access featureprofile repo.